memory/level_up_manager.py

```diff
@@ -0,0 +1,123 @@
+from enum import Enum
+from typing import Self
+
+from memory.core import mem_handle
+from memory.mappers.player_party_character import PlayerPartyCharacter
+
+
+class LevelUpUpgradeType(Enum):
```

Comment:
These map directly onto memory values in the game or?

Reply:
Yes